v2/account_claims.go

```diff
@@ -72,26 +71,36 @@ func (j *JetStreamLimits) IsJSEnabled() bool {
 	return j.MemoryStorage != 0 || j.DiskStorage != 0
 }
 
+type TieredLimits map[string]JetStreamLimits
+
 // OperatorLimits are used to limit access by an account
 type OperatorLimits struct {
 	NatsLimits
 	AccountLimits
 	JetStreamLimits
+	TieredLimits `json:"tiered_limits,omitempty"`
```

Comment:
Since these are ultimately JetStream limits not sure why don't we group them with JetStreamLimits.

Reply:
so we don't get recursive tiers. 
It's either move tieredlimits into JetStream Limits but then the actual values have to be moved into their own struct.
Or keep them next to each other.